common/beerocks/bcl/include/beerocks/bcl/beerocks_defines.h

```diff
@@ -26,7 +26,7 @@ namespace beerocks {
 #define BEEROCKS_PLAT_MGR_UDS "uds_platform_manager"
 #define BEEROCKS_BACKHAUL_MGR_UDS "uds_backhaul_manager"
 
-#if __GNUC__ > 4
+#if __GNUC__ >= 7
 #define FALLTHROUGH [[fallthrough]]
```

Comment:
Actually, if we're anyway using a define for it, and it's done in a gcc condition, we could just as well use `__attribute__((fallthrough))` which has been supported in GCC since basically forever. So the condition could just be `#if __GNUC__`.

Yes, `[[fallthrough]]` is actually standardized, but within a `#if __GNUC__` that doesn't matter much.

Reply:
I'm not sure I understand what I suppose to do here.

Reply:
fixed